src/monaco/monaco.ts

```
@@ -0,0 +1,24 @@
+import * as monaco from "monaco-editor";
```

ESLint

[error] 'monaco' is defined but never used.

(@typescript-eslint/no-unused-vars)